Testscripts/Linux/utils.sh

```diff
@@ -3817,4 +3817,22 @@ function is_hpc_vm() {
 	else
 		return 2
 	fi
-}
\ No newline at end of file
+}
+
+# Return the string from the dmesg, messages or syslog
+# Depending on distro, system logs are different.
+# if found, return 1. Otherwise, 0.
+function found_sys_log() {
+	if [ -f /var/log/messages ]; then
+		_ret=$(sudo cat /var/log/messages | grep -i "$1")
+	elif [ -f /var/log/syslog ]; then
+		_ret=$(sudo cat /var/log/syslog | grep -i "$1")
+	else
+		_ret=$(sudo dmesg | grep -i "$1")
+	fi
+	if [[ "$_ret" == *"$1"* ]]; then
```

Comment:
I think we can avoid this check, by depending on what the error code grep command returns. As we are doing same string matching 2 times with the same string.
Let me know your views on this.